### ISSUE TITLE
manage devices is very hard to find

### ISSUE THREAD

**Issue by rhaas80** [cilogon/access-identity#29]
The MFA help page [https://identity.access-ci.org/manage-mfa](https://identity.access-ci.org/manage-mfa) just points to the generic DUO page on the universal login prompt [https://guide.duo.com/universal-prompt#add-another-device](https://guide.duo.com/universal-prompt#add-another-device) where one has to first search how to manage devices. 

Having learned that one has to do so on the actual login prompt it turns out that it is quite hard to get to that prompt if one is currently logged onto ACCESS-CI's idm. Just using the "Logout" button in the "My Access" menu item is not enough since it does not actually logs one out and no prompt is shown if one clicks on "login" afterwards. Instead one has to know the "secret" URL [https://cilogon.org/logout](https://cilogon.org/logout) to be offered an option by CILogon to fully log out of ACCESS.

This seems to be unnecessarily complicated.

**Comment by jbasney** [comment]
Thanks @rhaas80 for the feedback. Duo's new universal prompt seems to have made this more difficult. Also, we've increased the single sign-on session length to 18 hours, so the Duo prompt appears less frequently.

I've updated https://identity.access-ci.org/manage-mfa (a5195bc540a313a8e252509ca79c13924048c244) with step-by-step instructions to reach the Duo device management page using a private browser window.
